git_machete/client.py

```diff
@@ -1670,14 +1670,16 @@ def checkout_github_pr(self, pr_no: int) -> None:
         pr: GitHubPullRequest = find_pr_by_number(pr_no, all_prs)
 
         if not pr:
-            raise MacheteException(f"PR number {pr_no} is not found in repository {repo}")
+            raise MacheteException(f"PR #{pr_no} is not found in repository {org}/{repo}")
         debug('checkout_github_pr()', f'found {pr}')
 
         print(f"Fetching {remote}...")
         self.__git.fetch_remote(remote)
+        if self.__git.remotes():
+            self.flush_caches()
```

Comment:
Let's leave as is for now, but now I see this requires a more systematic approach: pls add a TODO here for https://github.com/VirtusLab/git-machete/issues/237